dts/msm8916/msm8916-vivo-y21l.dts

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+
+/ {
+    qcom,msm-id = <0xce 0x00 0xf8 0x00 0xf9 0x00 0xfa 0x00>;
```

Comment:
```suggestion
	qcom,msm-id = <206 0>;
```

Unless it has another non-LTE variant or doesn't load the device tree,
otherwise it's fine to keep only `<206 0>`/`<0xce 0x0>`

Additional info:
- msm8216 (0xf8/248)
- msm8116 (0xf9/249)
- msm8616 (0xfa/250)

Ref: https://github.com/msm8916-mainline/lk2nd/blob/experimental-tmp3/lk2nd/include/dt-bindings/arm/qcom%2Cids.h#L106-L108